lib/webpacker/helper.rb

```diff
@@ -7,8 +7,8 @@ def current_webpacker_instance
   end
 
   # Computes the relative path for a given Webpacker asset.
-  # Returns the relative path using manifest.json and passes it to asset_path helper.
-  # This will use asset_path internally, so most of their behaviors will be the same.
+  # Returns the relative path using manifest.json and passes it to path_to_asset helper.
+  # This will use path_to_asset internally, so most of their behaviors will be the same.
```

Comment:
I dithered on whether to rename the method in the comments, too, since `path_to_asset` is probably not as well known as `asset_path`. I thought it was better to be consistent.